watch-and-repack.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Relative paths to the web repositories
+PENUMBRA_ZONE_WEB_PATH="../../web"
```

Comment:
devs should modify this relative path

Reply:
instead of unconditionally clobbering this var, check it if exists and is a valid dir using `readlink`

Reply:
modified to check env vars

Reply:
to be more clear - i think it is better to fail entirely if the test fails. the correct location will be incredibly user-specific, and also this default is also not checked the same way as an incoming var, so is likely to fail anyway.

Reply:
ah good point, modified to echo an error message and exit if unset.